### ISSUE TITLE
Multiple flaws with floating connection windows

### ISSUE THREAD

**Issue by farosch** [mRemoteNG/mRemoteNG#1308]
- [ ] Floating window is always on top of mrng main form
- [ ] Closing a floating window running a RDP session quits the running session but does not close the window itself
- [ ] Resizing a floating window or the main form running a RDP session does not resize the remote desktop
- [ ] Unable to "dock" floating window by dragging and dropping to desktop edges
- [ ] Floating window lacks minimize/maximize buttons

![ezgif-4-6775bb521a07](https://user-images.githubusercontent.com/22905106/52730414-94011700-2fbb-11e9-9f35-320794595682.gif)

## Your Environment
* Version used: latest develop branch
* Operating System and version: Windows 10 1809 17763.316 x64

**Comment by farosch** [comment]
@jotatsu Could you maybe take a look at these?

**Comment by jotatsu** [comment]
@farosch Yep, i'm looking at this ones , FloatWindowNG is already extended from DPS, just need to add some more customization.

**Comment by magriggs** [comment]
Please assign to me

**Comment by magriggs** [comment]
Status:

Floating window is always on top of mrng main form: FIXED
Closing a floating window running a RDP session quits the running session but does not close the window itself: FIXED
Resizing a floating window or the main form running a RDP session does not resize the remote desktop: TBC
Unable to "dock" floating window by dragging and dropping to desktop edges: TBC 
Floating window lacks minimize/maximize buttons: FIXED
Floating windows do not appear on the taskbar or when using Alt+Tab: FIXED

**Comment by Kvarkas** [comment]
PR please and will approve! Thanks
